aggregate/all_accessors.py

```diff
@@ -0,0 +1,71 @@
+# Housing production imports
+from aggregate.housing_production.area_within_historic_district import (
+    fraction_historic,
+)
+from aggregate.housing_production.change_in_units import change_in_units
+from aggregate.housing_production.hpd_housing_ny_affordable_housing import (
+    affordable_housing,
+)
+
+# Housing Security imports
+# Currently count residential evictions is left out as it requires geocoding, can address this later if need be.
+# from aggregate.housing_security.three_or_more_maintenance_deficiencies import (
+#     count_units_three_or_more_deficiencies,
+# )
+
+# Three or more maintenance deficiences also left out as I'm not sure where we are getting data moving forward
+# from aggregate.housing_security.evictions_by_city_marshals import (
```

Comment:
This will be rectified with Te's pull request #211

Reply:
yep!